src/codeu/chat/common/SinglesView.java

```diff
@@ -43,4 +44,10 @@
   // matches the given id, null will be returned.
   Message findMessage(Uuid id);
 
+  // FIND INTERESTS
+  //
+  // Find the interests for the user with the given id.
+  // TODO: uncomment when implemented in client View
```

Comment:
More TODOs!

Note that if you really can't do something yet, you should remove it completely before trying to push your code instead of commenting it out.